test/Interop/Cxx/templates/class-template-instantiation-error-handling.swift

```diff
@@ -0,0 +1,19 @@
+// RUN: not %target-swift-emit-sil %s -I %S/Inputs -enable-cxx-interop 2>&1 | %FileCheck %s
```

Comment:
Could you use `%target-swift-emit-sil -verify` (I think that should work...) instead of using `%FileCheck` here? We generally check diagnostics using `-verify`, not with `FileCheck`. If the errors are being emitting during SILGen, you could use `%target-swift-emit-silgen`.

Reply:
Is there a way to expect an error happening in a different file? `-verify` seems to assume the errors to happen in the current file, but in my case I want assert that the error shows location in the header where the instantiation happened.

Reply:
(Sorry, missed this comment earlier.)

> in my case I want assert that the error shows location in the header where the instantiation happened.

I see, I didn't realize that was happening. That makes sense.